CMakeLists.txt

```diff
@@ -389,7 +389,7 @@ define_pkg(devel
   GROUP "libs"
   DISPLAY_NAME "opae-libs"
   DESCRIPTION "OPAE runtime"
-  DEB_DEPENDS "uuid-dev , libjson0"
+  DEB_DEPENDS "uuid-dev , libjson-c-dev"
```

Comment:
For a runtime library (opae-libs), it should depend on runtime packages.
We should change uuid-dev to uuid but for json-c, we should be more explicit/deliberate  about the package.
While libjson-c-dev pulls in the correct runtime package, that is also pulling in more dependencies than are really needed.
It would be nice if we could use a CMake variable here instead.
Something like:
```
DEB_DEPENDS uuid ${json-c-pkg}
```

And hopefully, we could detect which distro we're on to set that variable (json-c-pkg).